crates/oxc_codegen/src/gen.rs

```diff
@@ -1093,6 +1095,24 @@ fn print_str<const MINIFY: bool>(s: &str, p: &mut Codegen<{ MINIFY }>) {
     p.print(b'\'');
 }
 
+fn print_origin_str<const MINIFY: bool>(s: &str, p: &mut Codegen<{ MINIFY }>) {
```

Comment:
@Boshen I notice the `print_str` is changed, add the function maybe has messed, Maybe we can escape str and also it isn't valid?

Reply:
Instead of print_str, maybe we can use the original string slice:

directive.span.source_text(self.source_text())

(We'll need to pass in the original source text to codegen).